src/config/options.rs

```diff
@@ -236,6 +236,9 @@ pub struct WidthHeuristics {
     // Maximum line length for single line if-else expressions. A value
     // of zero means always break if-else expressions.
     pub(crate) single_line_if_else_max_width: usize,
+    // Maximum line length for single line let-else statements. A value of zero means
+    // always format the divergent `else block` over multiple lines.
```

Comment:
Same theme as prior comments

```suggestion
    // always format the divergent `else` block over multiple lines.
```